model/technology.go

```diff
@@ -23,6 +23,8 @@ const (
 	TechnologyMariaDBFoundationMariaDB string = "MariaDBFoundation/MariaDB"
 	TechnologyPostgreSQLPostgreSQL     string = "PostgreSQL/PostgreSQL"
 	TechnologyOracleMySQL              string = "Oracle/MySQL"
+	TechnologyOracleOVM                string = "Oracle/OVM"
+	TechnologyVMWare                   string = "VMWare/VMWare"
```

Comment:
Are we sure that os VMware and not ESXi o vCenter?

Reply:
VMWare is the product;
ESXi is the OS for physical servers where VMs runs;
vCenter is the admin console;

Stakeholder told me it's ok to use VMWare as producer and VMWare as a product, so the technology name